pkg/codegen/schema/mock_pulumi_schema.go

```diff
@@ -38,8 +34,7 @@ func newPulumiPackage() *Package {
 	}
 
 	pkg, err := ImportSpec(spec, nil)
-	contract.AssertNoError(err)
```

Comment:
This really really shouldn't ever error, the assert is correct.

Reply:
This checks the user's `Pulumi.yaml` which can be incorrect.

This is then run before main because of the top level `var DefaultPulumiPackage = newPulumiPackage()` on line 45

Reply:
This is the core pulumi schema, not the users Pulumi.yaml

Reply:
This is the core pulumi schema, not the users Pulumi.yaml